x-pack/plugins/synthetics/server/synthetics_service/formatters/public_formatters/browser.ts

```diff
@@ -34,14 +38,26 @@ const throttlingFormatter: Formatter = (fields) => {
 };
 
 export const browserFormatters: BrowserFormatMap = {
-  ...basicBrowserFormatters,
+  ...commonFormatters,
+  ...tlsFormatters,
+  [ConfigKey.SOURCE_PROJECT_CONTENT]: null,
+  [ConfigKey.SCREENSHOTS]: null,
+  [ConfigKey.IGNORE_HTTPS_ERRORS]: null,
+  [ConfigKey.PLAYWRIGHT_OPTIONS]: null,
+  [ConfigKey.TEXT_ASSERTION]: stringToJsonFormatter,
+  [ConfigKey.PORT]: stringToJsonFormatter,
```

Comment:
Should not be importing `stringToJsonFormatter` into public, as well as `arrayToJSON` and `objectToJSON`